src/js/components/Select/stories/Basics.js

```diff
@@ -39,6 +40,10 @@ const customRoundedTheme = deepMerge(grommet, {
         border: '1px solid #7D4CDB',
       },
     },
+    icons: {
+      down: FormDown,
+      up: FormUp,
+    },
```

Comment:
According to chromatic, this added theme is flipping the up/down logic https://www.chromaticqa.com/build?appId=5d9774839a6eff00203f5cbf&number=1688

Reply:
Chromatic doesn't seem to be picking up that the Select is open by default in that story. The up/down logic is actually correct. Prior, this story allows showed a down arrow, but since this story shows the Select open by default, we would expect it to show the up arrow. 

If you open this story in the deployment, you'll see it's correct. I'm not sure why chromatic doesn't show that the items are displaying: https://deploy-preview-3950--sad-tereshkova-173d07.netlify.com/?path=/story/select--custom-theme